modules/check_in/app/services/travel_claim/claim_submission_service.rb

```diff
@@ -114,6 +118,10 @@ def validate_appointment_date_format
       end
     end
```

Comment:
taking a page out of the [SMOC service](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/travel_pay/app/services/travel_pay/date_utils.rb) for date-time normalization and validation